vmware/cloud-provider-for-cloud-director
# [VKCI-65] Disable service controller if network is not backed by NSX-T

Signed-off-by: Aniruddha Shamasundar <aniruddha.9794@gmail.com>
Service controller needs to be disabled if the orgvdc network is not backed by NSX-T. This makes CPI to only work as a node controller when network is not NSX-T backed.

Testing done:
- Create CPI  with an NSX-T backed ovdc network - Service controller starts up and handles load balancer CRUD
- Create CPI with a direct ovdc network - Service controller should be disabled. (as shown in the screenshot below)
![Screen Shot 2022-12-16 at 1 59 46 PM](https://user-images.githubusercontent.com/16699642/208196063-7dfa98cc-8e26-461c-b840-bba11dbba396.png)



- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vmware/cloud-provider-for-cloud-director/194)
